assets/scss/components/_buttonNeue.scss

```diff
@@ -0,0 +1,323 @@
+$C_accent: rgba(241,58,89,1);
```

Comment:
Sorry, should have been more clear; I was wondering if this entire file was intended to be committed